CHANGES.rst

```diff
@@ -392,6 +392,9 @@ Bug fixes
 
 - ``astropy.coordinates``
 
+  - Fix string representation of ``SkyCoord`` objects transformed into
+    the ``AltAz`` frame
```

Comment:
Actually--while you're at it--this fix affects the v1.0.x release so this should be a bug fix in Astropy v1.0.5
However, when you first started this PR, Astropy v1.0.4 wasn't out yet.  So you'll have to rebase onto master in order for the correct section in the changelog to appear.  Then move this to the correct spot.  Sorry.